### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by hoanganhtuanbk**

### CODE REVIEW

File: App/Containers/ContactsAddScreen.js [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+            <ActivityIndicator size="large" color={Platform.OS === 'ios' ? 'white' : '#EA0000'} />
+          </View>
+        }
+        <ScrollView style={[styles.mainContainer, {
+          height: Metrics.screenHeight,
+          marginBottom: 60
+        }]}>
+          <KeyboardAvoidingView behavior="padding">
+            <Form
+              ref="form"
```
Comment (schremsn) [inline App/Containers/ContactsAddScreen.js]: string refs are deprecated
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-string-refs.md

File: App/Containers/ContactsAddScreen.js [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+  View, ActivityIndicator, Image, KeyboardAvoidingView, ScrollView, ToastAndroid,
+  Platform
+} from 'react-native';
+import I18n from 'react-native-i18n';
+import { Images, Colors, Metrics } from './../Themes';
+import styles from './Styles/ContainerStyles';
+import Header from '../Components/Header';
+import { createCustomer } from '../Redux/ContactsRedux';
+import RoundedButton from '../Components/RoundedButton';
+import t from 'tcomb-form-native'
```
Comment (schremsn) [inline App/Containers/ContactsAddScreen.js]: check for a bunch of eslint issues, e.g. missing semi-colon

File: App/Containers/ContactsAddScreen.js [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+import  _ from 'lodash'
+const stylesheet = _.cloneDeep(t.form.Form.stylesheet);
+
+stylesheet.controlLabel.normal.color = 'white';
+stylesheet.textbox.normal.color = 'white';
```
Comment (schremsn) [inline App/Containers/ContactsAddScreen.js]: why are you setting this locally?

File: App/Containers/ContactsAddScreen.js [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+
+const Form = t.form.Form;
+const Email = t.refinement(t.String, function (text) {
```
Comment (schremsn) [inline App/Containers/ContactsAddScreen.js]: consider using an existing component, e.g. https://www.npmjs.com/package/isemail

File: App/Containers/ContactsAddScreen.js [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+  constructor() {
+    super();
+    this.state = {
+      value: {
+        is_company: true
+      },
+      isLoading: false
+    };
+    this.onChange = this.onChange.bind(this)
+    this.onPress = this.onPress.bind(this)
```
Comment (schremsn) [inline App/Containers/ContactsAddScreen.js]: eslint issues

File: App/Containers/ContactsAddScreen.js [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+      this.setState({ isLoading: true})
+      createCustomer(value)
+        .then((res) =>{
+          this.setState({ isLoading: false})
+          ToastAndroid.show(I18n.t('Create contacts is success'), ToastAndroid.SHORT);
+          this.props.navigation.replace('ContactsListScreen')
+        })
+        .catch(error =>{
+          this.setState({ isLoading: false})
+          ToastAndroid.show(error, ToastAndroid.SHORT);
```
Comment (schremsn) [inline App/Containers/ContactsAddScreen.js]: for an error an alert is more appropriate, so the user does not miss
eslint issues as well

File: App/Containers/ContactsAddScreen.js [hunk 7]
```diff
@@ -0,0 +1,178 @@
...
+        {
+          isLoading && <View style={[styles.progressBarLoading]}>
+            <ActivityIndicator size="large" color={Platform.OS === 'ios' ? 'white' : '#EA0000'} />
+          </View>
+        }
+        <ScrollView style={[styles.mainContainer, {
+          height: Metrics.screenHeight,
+          marginBottom: 60
+        }]}>
+          <KeyboardAvoidingView behavior="padding">
```
Comment (schremsn) [inline App/Containers/ContactsAddScreen.js]: this is not working to well.
Note: Android and iOS both interact with this prop differently. Android may behave better when given no behavior prop at all, whereas iOS is the opposite.
i have not had a chance to see which prop works best

File: App/Containers/ContactsAddScreen.js (hunk 1 above)
Comment (hoanganhtuanbk) [reply App/Containers/ContactsAddScreen.js]: I got it. I fixed it.

File: App/Containers/ContactsAddScreen.js (hunk 2 above)
Comment (hoanganhtuanbk) [reply App/Containers/ContactsAddScreen.js]: I reInstall eslint airbnb and it was fixed.

File: App/Containers/ContactsAddScreen.js (hunk 3 above)
Comment (hoanganhtuanbk) [reply App/Containers/ContactsAddScreen.js]: i updated it. I moved it to global styles file.

File: App/Containers/ContactsAddScreen.js (hunk 4 above)
Comment (hoanganhtuanbk) [reply App/Containers/ContactsAddScreen.js]: Tks, I updated it.

File: App/Containers/ContactsAddScreen.js (hunk 7 above)
Comment (hoanganhtuanbk) [reply App/Containers/ContactsAddScreen.js]: I will test this issue on ios device.

**Pull request merged**